src/main/groovy/net/minecraftforge/gradleutils/GradleUtils.groovy

```diff
@@ -175,88 +171,86 @@ class GradleUtils {
         }
     }
 
+    // TODO: [GradleUtils] Make language-agnostic by changing to Action<? super MavenArtifactRepository> in GU 3.0
     /**
```

Comment:
since all of our consumers are groovy, and thus runtime compiled, do we need to wait with this TODO?

Reply:
Good call. I'm pretty sure all consumers of GradleUtils are the buildscripts, but I wasn't entirely sure at the time I made this TODO.